metadata.yaml

```diff
@@ -42,3 +42,6 @@ requires:
     interface: microk8s-info
     scope: global
     limit: 1
+  dns:
```

Comment:
I thought you said you wanted to change this to `dns-provider` to match `kubernetes-control-plane` and `coredns` charm

Reply:
Yeah, that was breaking things for some reason. I'll update these once everything works fine.

Reply:
@sachinkumarsingh092 Any updates on this?